### PR TITLE
feat: Add a debug flag for request logs, scrub logs for apiAccessToken

### PR DESCRIPTION
**Pull request by toorooji**
# Title

Add a debug flag for logs, scrub logs for sensitive data

## Description

Request/response debugging will now be disabled by default. To enable, update the debug flag in abcConfig.json. Additionally, we're now removing sensitive data from those logs.

## Motivation and Context

The logs are not necessary in production mode, so this should cause no customer impact. Additionally, this will allow the developer to decide when to enable logging.

## Testing

Created a new skill, tested using the simulator, deployed to a test environment to view logs.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [X] My code follows the code style of this project
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly
- [X] I have read the **README** document
- [ ] I have added tests to cover my changes
- [X] All new and existing tests passed

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

[issues]: https://github.com/alexa-games/skill-flow-builder/issues

### CODE REVIEW

File: packages/sfb-cli/samples/adventure/code/index.ts [hunk 1]
```diff
@@ -28,7 +28,11 @@ const configAccessor = new ConfigAccessor(require(path.resolve("abcConfig", "abc
  * Skill handler (Request Entry Point)
  */
 export async function handler(event: any, context: any, callback: any) {
-	console.log(`[INFO] Request Received: ${JSON.stringify(event, null, 4)}`);
+    const debug = configAccessor.isDebug();
+
+    if(debug)
```
Comment (anfazio) [inline packages/sfb-cli/samples/adventure/code/index.ts:33]: Can we add curly braces to the `if(debug)` blocks added in this PR?

File: packages/sfb-cli/sfb-cli.iml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (anfazio) [inline packages/sfb-cli/sfb-cli.iml]: These *.iml files don't need to be checked in. Maybe add *.iml to the .gitignore as well?

File: packages/sfb-editor/~/testing-temp/SFB/SlotTypes.json [hunk 3]
```diff
@@ -0,0 +1 @@
+{}
```
Comment (anfazio) [inline packages/sfb-editor/~/testing-temp/SFB/SlotTypes.json]: Remove these `testing-temp` files

File: package.json [hunk 4]
```diff
@@ -44,5 +44,8 @@
     "node": ">=10.16.0",
     "npm": ">=6.x",
     "yarn": ">=1.x"
+  },
+  "dependencies": {
+    "@alexa-games/sfb-cli": "/Users/toorooji/workspace/skill-flow-builder/packages/sfb-cli/"
```
Comment (anfazio) [inline package.json]: This probably shouldn't be included

**Review by anfazio — COMMENTED** [review]
A few minor changes, but otherwise looks good

File: packages/sfb-cli/sfb-cli.iml (hunk 2 above)
Comment (toorooji) [reply packages/sfb-cli/sfb-cli.iml]: Haha, you're speedy. I was removing all of these as you were commenting :D

File: packages/sfb-cli/samples/adventure/code/index.ts (hunk 1 above)
Comment (toorooji) [reply packages/sfb-cli/samples/adventure/code/index.ts:33]: Will update now.

File: package.json (hunk 4 above)
Comment (toorooji) [reply package.json]: Sorry, removed after but you go to it first :D

**Review by anfazio — APPROVED** [review]

**Pull request merged**